### ISSUE TITLE
The app only accepts one payer per expense

### ISSUE THREAD

**Issue by fferreira12** [fferreira12/expenses-splitter#16]
The app currently only supports 1 payer per expense, meaning he/she will pay for the total of the expense.
It would be nice if the app accepted multiple payers per expense, with variable amounts per user.
It would also be nice to have an option to split evenly among all payers (which may be different from all participants in an expense)

**Comment by fferreira12** [comment]
This feature was added by c1e4e7bb18523bf416dd37a557652251e1fae273.
At home I had a bug where on page refresh the expense list did not load correctly and the app was recursively calling the expense creation methods.
I could not reproduce this at work.
Must see if problem persists at home or shows up at work.
Also, I had to use a workaround at the expense list component since the expense method (isPayer) was not being found on the expenses after page refresh.
